labs/lab6/lab6.adoc

```diff
@@ -29,56 +37,311 @@ Password: INSTRUCTOR WILL PROVIDE
+
+Execute the following command to access the Prometheus container:
+
+.master$
+[source, bash]
+----
+oc rsh -c prometheus $(oc get pods -l=app=prometheus -o 'jsonpath={.items[0].metadata.name}')
+----
+
+Once inside the container, confirm the file mounted in the ConfigMap has been updated.
+
+.prometheus container$
+[source, bash]
+----
+cat /etc/prometheus/prometheus.additional.rules
+----
+
+It may take up to 1 minute for the value to change. Continue executing the prior command until a change is detected.
+
+Once the value in the files has changed, to force Prometheus to reload its configuration, execute the following request
+
+.prometheus container$
+[source, bash]
+----
+curl -X POST http://localhost:9090/-/reload
+----
+
+To confirm the configuration was reloaded properly, revisit the Alerts page within Prometheus and reload the page. The updated value of the rule should be showing which will trigger an alert if the number of application nodes is less than 2 instances.
+
+Prometheus scrapes the OpenShift API once every 60 seconds. After the interval has passed, refresh the page again and the alert should turn _yellow_. This indicates the alert is pending:
+
+image::images/alertmanager-yellow.png[]
+
+If you recall in the rule, a _for_ clause was provided which defines that a rule will continued to be checked for a set period of time before firing. This allows a condition to resolve itself without a rule firing (such as a temporary network issue). The alert will stay in this condition for 2 minutes, then the alert wil fire. Refresh the page to see the status of the rule in _FIRING_ status:
+
+image::images/alertmanager-red.png[]
+
+The series of actions described previously should have been executed:
+
+1. Prometheus notifies AlertManager
+2. AlertManager parses the alert, passes it to the receiver and invokes the AlertManager Tower Bridge application
+3. The AlertManager Tower Bridge Application parses the alert and invokes the Workflow Job in Ansible Tower
+4. The Ansible Tower Workflow job runs to completion.
+
+Let's validate that this occurred properly.
+
+First, navigate to the AlertManager console or the tab that was previously open.
+
+link:https://alertmanager-openshift-metrics.apps-<student_id>.labs.sysdeseng.com[https://alertmanager-openshift-metrics.apps-<student_id>.labs.sysdeseng.com]
+
+An alert should now be indicated on the _Alerts_ page as shown below.
+
+image::images/alertmanager-console.png[]
+
+Notice how the `type=ansible-tower` is prominently displayed.
+
+Now, verify the AlertManager Tower Bridge application has received the webhook and invoked Ansible Tower.
+
+Navigate to the OpenShift web console and login using the _<student_id>-admin_ account using the provided credentials.
+
+link:https://master-<student_id>.labs.sysdeseng.com:8443[https://master-<student_id>.labs.sysdeseng.com:8443]
+
+On the righthand side of the page, select the _openshift-metrics_ project. 
```

Comment:
It depends on where they are.  I was already in the openshift metrics project.  Maybe something like: Return to the openshift-metrics project if you aren't there already.